addons/mail/static/tests/messaging_menu/messaging_menu_tests.js

```diff
@@ -74,7 +74,7 @@ QUnit.test("rendering with OdooBot has a request (default)", async (assert) => {
     assert.ok(
         $(".o-mail-NotificationItem img")
             .data("src")
-            .includes("/web/image?field=avatar_128&id=2&model=res.partner")
+            .includes("/web/image/res.partner/2/avatar_128")
```

Comment:
prefer full assert rather than includes (I want to see the unique key in all tests)